client/src/Movies/Movie.js

```diff
@@ -25,17 +26,18 @@ export default class Movie extends Component {
         console.error(error);
       });
   };
+
   // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  componentWillReceiveProps(newProps){
```

Comment:
i know this was already setup in the project, but I just found out that `componentWillReceiveProps()` has been deprecated and there is a new pattern for this logic.
more info here >> https://reactjs.org/docs/react-component.html#updating
and here >> https://alligator.io/react/get-derived-state/